src/git/jbuild

```diff
@@ -1,7 +0,0 @@
-(jbuild_version 1)
```

Comment:
I will switch to `jbuilder` (and learn in the same time how to use it). I use `ocamlbuild` just because it's more convenience for me 😄 !